lib/vagrant-registration/plugin.rb

```diff
@@ -16,8 +16,14 @@ class Plugin < Vagrant.plugin('2')
       class << self
         def register(hook)
           setup_logging
-          hook.after(::Vagrant::Action::Builtin::SyncedFolders,
-                     VagrantPlugins::Registration::Action.action_register)
+
+          if vbguest_plugin?
+            hook.before(VagrantVbguest::Middleware,
+                                 VagrantPlugins::Registration::Action.action_register)
```

Comment:
I would always prepend the middleware names with `::` (or not but lets be consistent) and format the second line the same as before...